### DISCUSSION TITLE
ESLint and Prettier Issues with Svelte Files in New Project - Errors Not Displaying in Problem Console

### DISCUSSION THREAD

**Discussion by axadrn** [sveltejs/kit#12591] [Q&A] [answered]
Hi everyone,

I'm facing an issue with a brand-new SvelteKit project that I’ve set up with ESLint and Prettier. After some tweaking with the new FlatConfig setup (had to make the config file visible), I got ESLint working perfectly in Neovim. Errors and warnings show up correctly in both TypeScript and JavaScript files, as well as in `.svelte` files.

However, in VSCode, I'm running into problems specifically with `.svelte` files. While errors and warnings (like unused variables) are highlighted correctly in `.ts` and `.js` files and listed in the Problem console, they are not for `.svelte` files. The errors only appear if I hover over the specific piece of code, but they are not underlined in red or listed in the Problem console, making it difficult to track issues.

Has anyone else encountered this issue in VSCode? Or do you have any suggestions on how to resolve it? Everything works fine in Neovim, so I’m pretty sure it’s something specific to VSCode. Any help would be greatly appreciated!

Thanks in advance!

**Comment by axadrn** [comment, marked as answer]
Solved, the answer is:
```
{
  "eslint.validate": ["javascript", "javascriptreact", "svelte"]
}
```

**Reply by renhyl** [reply]
had the same issue, as per above comment adding 
`"eslint.validate": ["javascript", "svelte"]` to User Settings json file in VSCode has fixed it! thanks!
